### ISSUE TITLE
Make box patterns as simple as mediaflux patterns

### ISSUE THREAD

**Issue by tashrifbillah** [PREDICT-DPACC/lochness#23]
Current box patterns:
https://github.com/PREDICT-DPACC/lochness/blob/master/docs/source/configuration_file.rst#box-base

Mediaflux has improved upon them by making them simpler:
https://github.com/PREDICT-DPACC/lochness/blob/30dc646ac91ef3463c04c3fb7e35887221762ef8/lochness/mediaflux/__init__.py#L73

**Comment by kcho** [comment]
This is great Tashrif. Would you mind working on the `box` module to apply your improvements to the `lochenss.box` as well please?

**Comment by tashrifbillah** [comment]
Did PR #26 already finish this work? Or is it being done in PR #27?

**Comment by kcho** [comment]
Yes it's been finished in one of the previous PR. I just attempted to find where this update happened, but it's not in #26 nor #27. It must happened before #26.

These are the lines

https://github.com/PREDICT-DPACC/lochness/blob/52103a14b8d56019bf25d64247264946435db16a/lochness/box/__init__.py#L361-L368
